extensions/cache/src/main/java/org/radargun/stages/cache/background/BackgroundStressorLogic.java

```diff
@@ -19,7 +19,7 @@
  * @author Radim Vansa &lt;rvansa@redhat.com&gt;
  * @author Michal Linhard &lt;mlinhard@redhat.com&gt;
  */
-class LegacyLogic extends AbstractLogic {
+class BackgroundStressorLogic extends AbstractLogic {
```

Comment:
The other (LogLogic) also use 'stressors' to execute the requests; but be it if you wish.